mock/users.json

```diff
@@ -0,0 +1,45 @@
+[
+  {
+    "firstName": "Pero",
+    "lastName": "Peric",
+    "email": "pero@lumen.test",
+    "password": "$2a$10$WqX3o5bx/dm8qvB6CruYkeBkZIP1LH85fBk7bLJn.ptcZY09pqjm.",
+    "phone": "981234567",
+    "numberOfVisits": 0,
+    "numberOfLogins": 0,
+    "sector": "A-130",
+    "active": false,
+    "suspended": false,
+    "regDate": "01-01-2000",
+    "lastVisit": "01-01-2000"
```

Comment:
Maybe the dates could be something more realistic?

Reply:
Fixed :)